CHANGELOG.md

```diff
@@ -5,7 +5,11 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## Unreleased
+### Added
```

Comment:
If this is a new feature/minor, do we need to bump the package.json version as well?

Reply:
Good point.

Reply:
done